front/src/components/track-file-info.tsx

```diff
@@ -39,11 +42,19 @@ const TrackFileInfo = ({ trackId }: { trackId: number }) => {
 	const { t } = useTranslation();
 	const track = useQuery((id) => API.getTrack(id, ["song"]), trackId);
 	const sourceFile = useQuery(API.getSourceFile, track.data?.sourceFileId);
+	const songType = track.data?.song?.type;
 
 	const tableContent: Partial<
-		Record<TranslationKey, string | number | undefined>
+		Record<TranslationKey, string | number | undefined | JSX.Element>
 	> = {
 		name: track.data?.name,
+		type: track.data?.type,
+		songType: songType && (
```

Comment:
There is an edge case to consider: the track may not have a song (e.g. for a documentary video).
If `track.data.song === null`, it should display `N/A` (see the BPM for an example)